src/App.vue

```diff
@@ -14,3 +16,31 @@ export default defineComponent({
 	components: { AppHeader, AppFooter },
 });
 </script>
+
+<style>
+html {
+	font-size: 62.5%;
+}
+
+@media only screen and (max-width: 75em) {
+	html {
+		font-size: 56.25%;
+	}
+}
+
+@media only screen and (max-width: 56.25em) {
+	html {
+		font-size: 50%;
+	}
+}
+
+@media only screen and (min-width: 112.5em) {
+	html {
+		font-size: 75%;
+	}
+}
+.app_root {
+	max-width: 120rem;
+	margin: auto;
+}
```

Comment:
shall we have a proper CSS file for that? I'm sure there is a proper way to customize antd styles and have a global style definitions to include.